pkgs/by-name/te/tetrio-desktop/package.nix

```
@@ -49,50 +52,38 @@ stdenv.mkDerivation rec {
     gtk3
   ];
 
-  libPath = lib.makeLibraryPath [
-    libGL
-    libpulseaudio
-    systemd
-  ];
-
   unpackCmd = "dpkg -x $curSrc src";
 
   installPhase = ''
     runHook preInstall
 
-    mkdir $out
+    mkdir -p $out/bin
     cp -r opt/ usr/share/ $out
-
-    mkdir $out/bin
-    ln -s $out/opt/TETR.IO/TETR.IO $out/bin/
+    ln -s $out/opt/TETR.IO/TETR.IO $out/bin/tetrio
 
     substituteInPlace $out/share/applications/TETR.IO.desktop \
-      --replace-fail "Exec=/opt/TETR.IO/TETR.IO" "Exec=\"$out/opt/TETR.IO/TETR.IO\""
+      --replace-fail "Exec=/opt/TETR.IO/TETR.IO" "Exec=tetrio"
```

Desktop file didn't work for me with this. (because tetrio is not on the path)
```suggestion
      --replace-fail "Exec=/opt/TETR.IO/TETR.IO" "Exec=\"$out/bin/tetrio\""
```

---

Weird, since `tetrio` is in `$out/bin`, it should have been added to the path, no ?
Anyway, I updated the PR. Let me know if it's better.

---

(The Desktop File specification doesn't add quotes around properties: https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#example)

---

Good to know, thanks :+1: